src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -79,6 +80,17 @@
    jruby-request/wrap-with-error-handling
    ring/wrap-params))
 
+(defn static-file-content-request-handler
+  "Build the main request handler for the /static_file_content endpoint"
```

Comment:
Nitpick about docstring: "build" is maybe a little unclear, instead you could say "Returns a function which is the main request handler ..."